CMSIS/NN/Include/arm_nn_types.h

```diff
@@ -32,6 +32,8 @@
 #ifndef _ARM_NN_TYPES_H
 #define _ARM_NN_TYPES_H
 
+#include <stdint.h>
```

Comment:
Why not include arm_math_types.h ? which is including cmsis_compiler.h which is including stdint.h

Reply:
I started off with that, but changed it as the file didn't include any vector register types and thought I'd include only what is necessary for this building block of a file. That was the reasoning behind not using arm_math_types.h here. I don't have a strong opinion in this case.